### ISSUE TITLE
Nullable analysis should be run if any nullable warnings are enabled

### ISSUE THREAD

**Issue by cston** [dotnet/roslyn#30862]
Warnings should be reported below the `#pragma`:
```c#
#nullable enable
class Program
{
    static void F(object x)
    {
    }

#nullable disable
    static void F1(object? y, object? z)
    {
        F(y);
#pragma warning restore 8604
        F(z); // warning: argument may be null
    }

    static void F2(object? w)
    {
        F(w); // warning: argument may be null
    }
}
```
See https://github.com/dotnet/roslyn/pull/30843#issuecomment-434513097.

**Comment by jcouv** [comment]
Subsumed by https://github.com/dotnet/roslyn/issues/36131
Closing
